modules/setting/sync.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package setting
+
+import "code.gitea.io/gitea/modules/log"
+
+// Sync represents configuration of sync
+var Sync = struct {
+	LockServiceType    string
+	LockServiceConnStr string
+}{
+	LockServiceType: "memory",
+}
+
+func parseSyncSetting() {
+	sec := Cfg.Section("sync")
+	Sync.LockServiceType = sec.Key("LOCK_SERVICE_TYPE").MustString("memory")
+	if Sync.LockServiceType != "memory" && Sync.LockServiceType != "redis" {
```

Comment:
Instead of an if conditional, please use a switch case so that in the future it can be expanded to more types
```
switch Sync.LockServiceType {
	case "memory":
	case "redis":
	default:
		log.Fatal("Unknown sync lock service type: %s", Sync.LockServiceType)
	}
```